pkg/ui/seeders.go

```diff
@@ -52,7 +52,8 @@ func (tui *TUI) SeedAckIncidentsUI() {
 	}
 
 	for _, i := range incidents {
-		incident := []string{i.Id, i.Title, i.Urgency, i.Status, i.Service.Summary}
+		I := i.Assignments[0]
```

Comment:
`I` for `Assingment` type variable naming is little confusing, 
```suggestion
		assignment := i.Assignments[0]
```

Reply:
Yes I have changed that, will update the PR once I implement supreeth's feature request.

Reply:
you can also click "accept suggestion" to use this code

Reply:
Thanks for the review, I have updated it.